### PR TITLE
[10.0][ADD] stock_lot_note: Add notes field to Lots

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: stock_lot_note/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    'name': 'Stock Lot Note',
+    'summary': """
+        Adds a field to fill in notes in lots""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Warehouse',
+    'maintainers': ['rousseldenis'],
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```
Comment (bodedra) [inline stock_lot_note/__manifest__.py]:
```suggestion
    'website': 'https://github.com/OCA/stock-logistics-warehouse',
```

**Review by bodedra — COMMENTED** [review]
Minor one.

File: stock_lot_note/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply stock_lot_note/__manifest__.py]: @bodedra Done

File: stock_lot_note/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Go to Inventory > Inventory Control > Lot/Serial Numbers
```
Comment (bodedra) [inline stock_lot_note/readme/USAGE.rst]:
```suggestion
Go to Inventory > Inventory Control > Lots/Serial Numbers
```

**Review by bodedra — APPROVED** [review]
Functional reviewed. :+1:

File: stock_lot_note/readme/USAGE.rst (hunk 2 above)
Comment (rousseldenis) [reply stock_lot_note/readme/USAGE.rst]: Done !

**Review by ps-tubtim — APPROVED** [review]
Code LGTM

**Pull request merged**